benches/search_comparison.rs

```diff
@@ -285,3 +257,17 @@ fn make_sorted_vec<T: Ord>(mut v: Vec<T>) -> Vec<T> {
 fn search_sorted_vec<'a, T: Ord>(c: &'a Vec<T>, x: T) -> Option<&'a T> {
     c.binary_search(&x).ok().map(|i| &c[i])
 }
+
+fn pseudorandom_iter<T>(mut seed: usize, max: usize) -> impl Iterator<Item = T>
```

Comment:
Let's remove first parameter. Not only the function is always used with `seed = 0`, but on the call site it's very easy to confuse the seed with the minimum value of generated numbers 😀

```rust
pseudorandom_iter(0, MAX)
```

Reply:
agree, removed seed